Examples/Python/src/Detector.cpp

```diff
@@ -108,6 +108,10 @@ void addDetector(Context& ctx) {
 
     auto c = py::class_<Config>(d, "Config").def(py::init<>());
 
+    c.def_property(
+        "jsonFile", nullptr,
+        [](Config& cfg, const std::string& file) { cfg.readJson(file); });
+
```

Comment:
Do I understand correctly that the reason for this is that you want to support passing `jsonFile` through the `create` interface?

Reply:
Yes. This seemed to be the easiest way to implement it, and allows the other configs to still be set in Python with the same `create` interface.